### PR TITLE
Fix use of <dfn value> and some indenting.

### LINKED ISSUE #218: Use of `value` definition type for terms that are not CSS values
The spec uses `value` definition types for some of its definitions, e.g. [`av01`](https://aomediacodec.github.io/av1-avif/#valdef-av1-image-item-type-av01).

It's tempting to use the `value` type for such values but, despite the generic name, that type was defined for **CSS** values, see [Bikeshed's documentation](https://speced.github.io/bikeshed/#dfn-types). The use of `value` confuses tools that parse specs to extract semantic information, because that makes them think that the spec defines CSS terms.

There is no better definition type than `dfn` for now for such values. The possiblity to create one is being tracked in https://github.com/speced/spec-dfn-contract/issues/6

### PR DESCRIPTION
**Pull request by leo-barnes**
This fixes #218.

@tidoust, please take a look so that I understood the issue correctly.

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/pull/223.html" title="Last updated on Sep 5, 2024, 8:25 AM UTC (b1e27f9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/223/8ebeb5f...b1e27f9.html" title="Last updated on Sep 5, 2024, 8:25 AM UTC (b1e27f9)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -168,17 +169,17 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
...
-  Box Type:                 <dfn value export for="AV1 Item Configuration Property">av1C</dfn>
+  Box Type:                 <dfn export for="AV1ItemConfigurationProperty">av1C</dfn>
   Property type:            Descriptive item property
   Container:                ItemPropertyContainerBox
   Mandatory (per  item):    Yes, for an image item of type 'av01'
   Quantity:                 One for an image item of type 'av01'
 </pre>
 
-The syntax and semantics of the <dfn>AV1 Item Configuration Property</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
+The syntax and semantics of the <dfn export>AV1ItemConfigurationProperty</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
```
Comment (y-guyon) [inline index.bs:182]: Should this be `[=AV1ItemConfigurationProperty=]` as it is already defined at line 175? What happens when there are multiple `<dfn>` for the same token? I looked at the [doc](https://speced.github.io/bikeshed/) but it was not clear.

File: index.bs [hunk 2]
```diff
@@ -423,17 +421,18 @@ The following additional constraints apply only to [=AV1 Image Sequences=]:
...
+    <code>avis, msf1, miaf, MA1A</code>
 </div>
 
-  <h2 id="mime-registration">AVIF Media Type Registration</h2>
+<h2 id="mime-registration">AVIF Media Type Registration</h2>
 
 <p>The media type <code>"image/avif"</code> is officially registered with IANA and available at: <a href="https://www.iana.org/assignments/media-types/image/avif">https://www.iana.org/assignments/media-types/image/avif</a>.</p>
 
-  <h2 id="change-list">Changes since v1.1.0 release</h2>
+<h2 id="change-list">Changes since v1.1.0 release</h2>
```
Comment (y-guyon) [inline index.bs:437]: Is this a list of changes in the source or in the generated document?

What does this PR do to the generated document?

**Review by y-guyon — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (leo-barnes) [reply index.bs:182]: I had to really dig into the BikeShed documentation and mess around with this to understand how it works. My understanding is that the definition on line 175 defines the term `AV1ItemConfigurationProperty/av1C` (i.e. `av1C` in the context of `AV1ItemConfigurationProperty`) as opposed to `av1C` as used in the sample entry in AV1-ISOBMFF. That is done to uniquely distinguish between the two uses.

The <dfn> on line 182 defines the term `AV1ItemConfigurationProperty` itself.

I tried changing line 182 to `[=AV1ItemConfigurationProperty=]`, but that causes BikeShed to complain about missing definitions for `AV1ItemConfigurationProperty`.

File: index.bs (hunk 2 above)
Comment (leo-barnes) [reply index.bs:437]: Ah. That is a good question. The generated document ends up _looking_ almost the same, but some of the defined terms in the document metadata are changed. So the Webref scraper will see different stuff for example.

The main noticeable difference for human readers is that `AV1 Item Configuration Property` is now `AV1ItemConfigurationProperty`.

Comment (leo-barnes) [reply index.bs:437]: I guess we need to figure out what the policy is regarding which changes end up in this list.

Comment (cconcolato) [reply index.bs:437]: You could list every PR and mark these kinds of PR as editorial.

**Comment by leo-barnes** [comment]
Apparently Bikeshed requires indentation to be tabs or 4 spaces for some stuff. I'm converting the whole document to using 4 spaces everywhere so we're consistent.

**Comment by leo-barnes** [comment]
Merging this so we can move forward with the other PR.s

**Comment by leo-barnes** [comment]
Argh. Should have squashed as well. I expected GitHub to do that automatically.

File: index.bs (hunk 2 above)
Comment (leo-barnes) [reply index.bs:438]: > You could list every PR and mark these kinds of PR as editorial.

I ended up doing this.

**Pull request merged**